src/base/bittorrent/sessionimpl.h

```diff
@@ -604,7 +604,7 @@ namespace BitTorrent
 
         void populateAdditionalTrackersFromURL();
 
-        std::vector<lt::alert *> getPendingAlerts(lt::time_duration time = lt::time_duration::zero()) const;
+        void getPendingAlerts(lt::time_duration time = lt::time_duration::zero());
```

Comment:
"fetchPendingAlerts" is better name for new signature.

Reply:
Done.